doc/0.120/_sources/feeds.rst.txt

```diff
@@ -0,0 +1,169 @@
+            <admarkt:description>This leather sofa is very well suited to sell online. Has happened 12 times
+            already, profit guaranteed! The charming white color and stylish design makes it irresistable
+            for buyers, whether they need it or not.
+            </admarkt:description>
+            <admarkt:categoryId>1234</admarkt:categoryId>
+            <admarkt:url>http://irresistiblecouch.com/charming-white.html?source=marktplaats</admarkt:url>
+            <admarkt:vanityUrl>http://irresistiblecouch.com</admarkt:vanityUrl>
+            <admarkt:price>129900</admarkt:price>
+            <admarkt:priceType>FIXED_PRICE</admarkt:priceType>
+            <admarkt:media>
+                <admarkt:image url="http://irresistiblecouch.com/ProductPhotos/Large/jh2fi82wqjckw.jpg"/>
+            </admarkt:media>
+            <admarkt:budget>
+                <admarkt:totalBudget>1000000</admarkt:totalBudget>
+                <admarkt:dailyBudget>2500</admarkt:dailyBudget>
+                <admarkt:cpc>3</admarkt:cpc>
+            </admarkt:budget>
+            <admarkt:emailAdvertiser>true</admarkt:emailAdvertiser>
+        </admarkt:ad>
+        <admarkt:ad>
+            <admarkt:vendorId>bvl202kfm2-123</admarkt:vendorId>
+            <admarkt:title>Nice bike</admarkt:title>
+            <admarkt:description>Decent women's bike. A little old, but good enough. Has seen some mileage,
+            been used regularly, but is well maintained. Last month had a large check-up where some key
+            parts were replaced.
+            </admarkt:description>
+            <admarkt:categoryId>43</admarkt:categoryId>
+            <admarkt:url>http://supabike.com/oldie.html?foo=bar</admarkt:url>
+            <admarkt:vanityUrl>http://supabike.com</admarkt:vanityUrl>
+            <admarkt:price>8900</admarkt:price>
+            <admarkt:priceType>FIXED_PRICE</admarkt:priceType>
+            <admarkt:media>
+                <admarkt:image url="http://supabike.com/files/colored/fiets.jpg"/>
+            </admarkt:media>
+            <admarkt:budget>
+                <admarkt:totalBudget>200000</admarkt:totalBudget>
+                <admarkt:dailyBudget></admarkt:dailyBudget>
+                <admarkt:cpc>2</admarkt:cpc>
+            </admarkt:budget>
+            <admarkt:emailAdvertiser>true</admarkt:emailAdvertiser>
+            <admarkt:microTip>Lease 499 per mnd</admarkt:microTip>
+        </admarkt:ad>
+    </admarkt:ads>
+
+
+Scenario's & Questions
+----------------------
+
+Below are some common asked-for scenarios and questions with their explanations/answers.
+
+- Feed file cannot be fetched
+
+When a feed file cannot be fetched, nothing will change on the user's ads.
+It's as if the import didn't happen. Since the file represents the desired
+list of ads to be live, we won't do anything if we can't get the file -
+we cannot read a change in the desired situation.
+
+- Feed file is empty
```

Comment:
Potentially -  ad a use-case here where images have changed but we do not see this change in ICAS

Reply:
that's there, at the bottom of the file. Do you want some more?

Reply:
Ah, I see it now sorry, missed it!